buildenv/jenkins/openjdk10_aarch64_linux

```diff
@@ -3,7 +3,8 @@
 	Set parameter TARGET(openjdk, system, external, perf, jck etc.).
 	Set UPSTREAM_JOB_NAME(version_build_arch_os, for example: openjdk8_build_x86-64_linux)
 	Set JVM_VERSION(openjdk8, openjdk8-openj9, openjdk9, openjdk8-openj9, etc.*/
-node('linux&&arm64&&test') {
+LABEL='linux&&arm64&&test'
```

Comment:
Using "def LABEL='linux&&arm64&&test" to define variable as a local one. Same for all others.

Reply:
`LABEL` will be used for parallelism in JenkinsfileBase, will this change has a side effect on that? Meanwhile, `LABEL` is the variable which defines which machine to run tests on, a global variable is making more sense since each Jenkinsfile is to used to support one specific platform. 

If this concern is about using a yml file to store labels, then labels and other variables will be return from a function and stored in a global variable. Did I miss any concerns?

Reply:
I personally  would not try to use global variable for it's difficult to know the side effect.